src/sentry/relay/config/experimental.py

```diff
@@ -76,3 +76,29 @@ def add_experimental_config(
         else:
             if subconfig is not None:
                 config[key] = subconfig
+
+
+R = TypeVar("R")
+
+
+def run_experimental_config_builder(
```

Comment:
I have chosen not to touch `add_experimental_config` because it would require far larger refactoring, and potentially break a lot of things, that is why new function was created.

I wasn't aware of the reasons behind the name, I will rename my function 😅